### DISCUSSION TITLE
want to solve waring [Failed to create bin at /app/node_modules/.bin/shadcn-ui. ENOENT: no such file or directory, open '/app/node_modules/shadcn-ui/dist/index.js']

### DISCUSSION THREAD

**Discussion by ak2-lucky** [shadcn-ui/ui#4968] [Q&A]
I always get a warning during `pnpm install`

```bash
root@b1ebb2dae028:/app# pnpm install
Lockfile is up to date, resolution step is skipped
Already up to date
 WARN  Failed to create bin at /app/node_modules/.bin/shadcn-ui. ENOENT: no such file or directory, open '/app/node_modules/shadcn-ui/dist/index.js'
 WARN  Failed to create bin at /app/node_modules/.bin/shadcn-ui. ENOENT: no such file or directory, open '/app/node_modules/shadcn-ui/dist/index.js'
Done in 3.2s
```

Clearing the cache and deleting node_modules didn't help!

This has been happening since I upgraded shadcn-ui from version 0.89 to 0.91.

How can this be resolved?

**Comment by ak2-lucky** [comment]
decide use "shadcn"

**Reply by magnobiet** [reply]
The shadcn-ui CLI is going to be deprecated soon. Use https://www.npmjs.com/package/shadcn instead.

**Reply by specious** [reply]
Basically, switch from [`shadcn-ui`](https://www.npmjs.com/package/shadcn-ui) to [`shadcn`](https://www.npmjs.com/package/shadcn).

**Comment by LukiRoth** [comment]
I’m encountering the same warning:

```
 WARN  Failed to create bin at \tauri-app\node_modules\.bin\shadcn-ui. ENOENT: no such file or directory, stat '\tauri-app\node_modules\shadcn-ui\dist\index.js.EXE'
 WARN  Failed to create bin at \tauri-app\node_modules\.bin\shadcn-ui. ENOENT: no such file or directory, stat '\tauri-app\node_modules\shadcn-ui\dist\index.js.EXE'
```

I’ve tried several steps to resolve this:

- Deleting `node_modules` and reinstalling using `pnpm install`
- Ensuring `shadcn-ui` is updated to the latest version (currently 0.9.2)
- Confirming that `pnpm` is up to date (version 9.12.1)

Additionally, I checked the following:
- There’s nothing named `shadcn-ui` in `\tauri-app\node_modules\.bin`.
- In `\tauri-app\node_modules\shadcn-ui`, there is no `dist` folder. Even after manually creating this folder, I still receive the same warning.

Does anyone have a solution for this issue?
